scan/tls/alert.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2009 The Go Authors. All rights reserved.
```

Comment:
This isn't a bad idea. To set it up as a submodule. I don't think we'll need to make any changes to the original `crypto/tls` files. I've just needed to access some functionality in creating the helper functions in `tls/cfsslscan_*`